backend/server/routes/index.js

```diff
@@ -19,6 +19,12 @@ router.route('/users/authenticate').post(user.authenticateUser)
 
 router.route('/users/:username/activity').get(user.getUserActivity)
 
+router.route('/interests').get(interests.getInterests)
+
+router.route('/interests').post(interests.addInterests)
+
+router.route('/interests').delete(interests.deleteInterests)
+
```

Comment:
```suggestion
router
  .route('/interests')
  .get(interests.getInterests)
  .post(interests.addInterests)
  .delete(interests.deleteInterests)

```
example (not sure if this is prettier)